js/ez-selection-activate-element.js

```diff
@@ -0,0 +1,56 @@
+(function () {
+    const SINGLE_SELECTION_CLASS = 'ez-js-activable-element-single-selection';
+    const ACTIVABLE_ELEMENT_CLASS = 'ez-js-activable-element';
+
+    /**
+     * `<ez-selection-activate-element>`
+     *
+     * @polymerElement
+     * @demo demo/ez-selection-activate-element.html
+     */
+    class SelectionActivateElement extends Polymer.Element {
+        static get is() {
+            return 'ez-selection-activate-element';
+        }
+
+        static get properties() {
+            return {
+                /**
+                 * The selectionSelector defines which checkboxes or radio buttons enable the activable element.
+                 */
+                selectionSelector: {
+                    type: String,
+                    reflectToAttribute: true,
+                    value: 'input[type="checkbox"]',
+                },
+            };
+        }
+
+        connectedCallback() {
+            super.connectedCallback();
+            this.addEventListener('click', this._updateActivableElements.bind(this));
+        }
+
+        /**
+         * Enables or disables the activable element depending on the checked checkboxes or radio buttons.
+         *
+         * @param {Event} e
+         */
+        _updateActivableElements(e) {
+            if (e.target.matches(this.selectionSelector)) {
+                const activableElements = this.querySelectorAll(`.${ACTIVABLE_ELEMENT_CLASS}`);
+                const checkedElementsNumber = this.querySelectorAll(this.selectionSelector + ':checked').length;
```

Comment:
use backticks instead of concatenating strings and this would also deserve to be done in a separated method